scripts/provision.sh

```diff
@@ -89,6 +89,7 @@ provider = "$OAUTH_PROVIDER"
 userinfo_url = "$OAUTH_USERINFO_URL"
 token_url = "$OAUTH_TOKEN_URL"
 redirect_url = "$OAUTH_REDIRECT_URL"
+signup_url = "$OAUTH_SIGNUP_URL"
```

Comment:
I believe this line is NOT needed as we don't have a signup_url config in builder-api at all

Reply:
Thats correct. My apologies @skylerto last week after we dug into this a bit more we found this has no effect so you can remove it and then we're g2g!

Reply:
awesome, just pushed a revert!